apps/docs/src/components/color/ColorTokens.tsx

```diff
@@ -1,9 +1,33 @@
-import { HvTypography, useTheme } from "@hitachivantara/uikit-react-core";
+import {
+  ds3,
+  ds5,
+  HvProvider,
+  HvTypography,
+  pentahoPlus,
+  useTheme,
+} from "@hitachivantara/uikit-react-core";
 
+import { useDocsTheme } from "../../hooks/useDocsTheme";
 import { descriptions } from "./descriptions";
 import { colorTokensSpec, groupColorTokensByCategory } from "./utils";
 
 export const ColorTokens = () => {
+  const { docsTheme, docsMode } = useDocsTheme();
+
+  return (
+    <HvProvider
+      themes={[pentahoPlus, ds5, ds3]}
+      theme={docsTheme}
+      colorMode={docsMode}
+      cssTheme="scoped"
+      rootElementId={"hv-uikit-docs-color-tokens"}
+    >
+      <ColorTokensInternal id="hv-uikit-docs-color-tokens" />
+    </HvProvider>
```

Comment:
we can reuse `DocsProvider`
```suggestion
    <DocsProvider className="space-y-10">
      <ColorTokensInternal />
    </DocsProvider>
```


if we change the container below to `<>`